### ISSUE TITLE
Finish new website

### ISSUE THREAD

**Issue by samanpwbb** [mapbox/maki-2#55]
Started nailing down the basic site architecture here: https://github.com/mapbox/maki-2/tree/website

Todo: 
- [ ] Switch to pre-rendering the react parts (@samanpwbb)
- [ ] Design the website (@natslaughter + @samanpwbb)
- [ ] Finish the styleguide (@natslaughter)

**Comment by natslaughter** [comment]
@samanpwbb and myself sat down and discussed the website's structure, flow, how users will use it and what it should look like. 

_How People Will Use It & Site Structure_

Users will visit the maki site to A: use the maki chef tool and download icons, and B: contribute to the maki icon set. We looked at two structure schemes: one that has everything on a single page and one that has separate pages for the maki chef tool and design guidelines ... plus a landing page. 

Scheme 1

This layout seems a bit too dense and although it is nice to have everything on one page sometimes, it might end up being confusing and not introducing the project well. There is a header area that introduces the project, along with a grid of maki icons to the right, or some sort of visual accompaniment. Below this is the maki chef editor (this might be renamed 'maki editor') and the design guidelines would follow this. At the bottom would be a footer (more on this later). We wanted to see what it would be like for the user to avoid a mouse click in order to get to maki chef. This might end up prioritizing maki chef over the design guidelines, and also doesn't have a flow that nicely presents the project. 

[maki-2-website-wireframes-scheme1.pdf](https://github.com/mapbox/maki-2/files/180374/maki-2-website-wireframes-scheme1.pdf)

Scheme 2

Separated into different pages, this scheme starts off with a landing page that introduces the project, and has basically two ways a user can enter the site; based on the two primary reasons why I think people will use the site. Both the 'style editor' and 'contribute' are introduced with some text, a visual element and lead the user to their respective pages. 

[maki-2-website-wireframes-scheme2.1.pdf](https://github.com/mapbox/maki-2/files/180380/maki-2-website-wireframes-scheme2.1.pdf)

[maki-2-website-wireframes-scheme2.2.pdf](https://github.com/mapbox/maki-2/files/180381/maki-2-website-wireframes-scheme2.2.pdf)

[maki-2-website-wireframes-scheme2.3.pdf](https://github.com/mapbox/maki-2/files/180382/maki-2-website-wireframes-scheme2.3.pdf)

_How It Should Look_

Looking at other sites that function similarly to this one, successful ones have little to no design embellishments to the site itself, and allow the major visual elements to be the user's work. Others have a lot of bells and whistles in the editor controls, and it competes too much with the user's work. 

Even though these are initial wireframe sketches, I am proposing that the site be very minimal in design, and not get in the way of the user's work. That said, in addition to being minimal, it should be very elegant and useful. 

There is also the question of whether or not the site should obey to the Mapbox branding guidelines, or how much it should adhere to them. Saman and I made an initial decision to see what it would look like not using the Mapbox branding elements, and treat it as a special project that can be a little different. Obviously, it needs to be connected to Mapbox, visually or not. If it ends up being visually distant to Mapbox's brand, the site needs to connect to the company in some other way. 

_A Fun Playful Moment (The Footer)_

Above I wrote the site should be very elegant and useful. I also think the site should have a moment of playfulness and character.

Being new to this project, I have enjoyed all of the little side stories connected to maki. I think they help tell the larger project story, and are fun to hear about. I am going to develop the footer to be comprised of vignettes that tell for example, why is it called maki?, how only successful icons look like faces, and other things I have heard. This is low on the priority list, but I think it would be nice for the site to show some of the character of the project and the people that have been involved. 

_Moving Forward_

I will continue to develop the site, looking at the primary elements such as the design guidelines layout, and the maki editor user interface.  More to come next week.
